src/main/resources/edu/cornell/kfs/concur/cu-ojb-concur.xml

```diff
@@ -33,4 +33,16 @@
     <field-descriptor name="validationResultMessage" column="VLDTN_RSLT_MSG" jdbc-type="VARCHAR" /> 
 </class-descriptor>
 
+<class-descriptor class="edu.cornell.kfs.concur.batch.businessobject.ConcurRequestedCashAdvance" table="CU_CNCR_REQ_CASH_ADV_T">
+    <field-descriptor name="concurRequestedCashAdvanceId" column="CNCR_REQ_CASH_ADV_ID" jdbc-type="INTEGER" primarykey="true" sequence-name="CNCR_REQ_CASH_ADV_ID_SEQ" autoincrement="true"/>
```

Comment:
Is the "INTEGER" JDBC type large enough for what you need, or do you need "BIGINT" instead?